src/modules/sensors/sensors.cpp

```diff
@@ -148,6 +148,7 @@ class Sensors : public ModuleBase<Sensors>, public ModuleParams, public px4::Sch
 
```

Comment:
Can you remove `_adc_fd` above?

Then it should also be possible to remove `ADC0_DEVICE_PATH` completely.

Reply:
Done.